test/externalTests/gnosis-v2.sh

```diff
@@ -30,12 +30,12 @@ function test_fn { npm test; }
 
 function gnosis_safe_test
 {
-    OPTIMIZER_LEVEL=1
+    OPTIMIZER_LEVEL=2
```

Comment:
Without the optimizer the code is too large to be deployed (triggers a warning about the Spurious Dragon limit).